### DISCUSSION TITLE
[PSD12] Exercise 3.6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#179] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210065007-1d4882c1-ea2d-4371-883e-098b2abedee4.png)

![image](https://user-images.githubusercontent.com/69389461/210065128-22f0382d-d0ed-4d83-90d1-84cde2b99a7b.png)

**Comment by Elia-Belli** [comment]
![DFF](https://user-images.githubusercontent.com/107944476/211634594-a763ca7e-0323-4fbe-b470-531e97383ffe.png)
Un D-flipflop campiona il segnale D solamente sul fronte (generalmente positivo) del clock

**Comment by fraxb04** [comment]
corretto
